tests/db_integrity/test_data_updates.py

```diff
@@ -809,7 +809,7 @@ def test_metadata_update():
             "TAC-185magl_UKV_EUROPE_TEST_201405.nc",
             "TAC-185magl_UKV_co2_EUROPE_TEST_201405.nc",
             "TAC",
-            "EUROPE",
+            "EUROPE-NON-STANDARD",
```

Comment:
Why does the domain need to change for this test? Is that so that alignment code is not called?

Reply:
I can chip in on this from a discussion yesterday - really in the tests we should only be using "EUROPE" for the domain name when we mean the exact "EUROPE" domain. So if there are cases with cut down domains we should be giving them a different name really.